### PR TITLE
Updated DataTables to v. 1.13.6 for qPCR pools

### PR DESCRIPTION
**Pull request by sofiahag**

### CODE REVIEW

**Comment by alneberg** [comment]
Looks good! Could we have the version of datatables in the file name? I think it makes it more transparent. Sorry for being picky

**Comment by sofiahag** [comment]
> Looks good! Could we have the version of datatables in the file name? I think it makes it more transparent. Sorry for being picky

Sure! No, that's smart! 👍🏻 🧠

**Comment by alneberg** [comment]
This looks good to me now, is it deployed on stage?

**Comment by sofiahag** [comment]
> This looks good to me now, is it deployed on stage?

It's up now! Though the queue pages always take forever to load, at least for me 🤷🏻‍♀️

**Comment by alneberg** [comment]
Forever as in that they never load?

**Review by alneberg — APPROVED** [review]
Great work!

**Comment by alneberg** [comment]
Let's deploy it on production and watch it carefully

File: run_dir/design/qpcr_pools.html [hunk 1]
```diff
@@ -49,7 +49,7 @@ <h1><span id="page_title">qPCR queues</span>
 </div>
 
 <script src="/static/js/clipboard.min.js"></script>
-<script src="/static/js/jquery.dataTables.min.js"></script>
+<script src="/static/js/dataTables-1.13.6.min.js"></script>
```
Comment (aanil) [inline run_dir/design/qpcr_pools.html:52]: The file name has a small t.

**Pull request merged**